Demo_Project/src/GlobalData/user.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+class USER extends React.Component<RouteComponentProps<{}>> {
+    constructor(props?:any,state?:any){
+        super(props,state)
+    }
+    public state = {
+        user:{}
+    };
+    setUSER({email,token,username,bio,image,id,createdAt,updatedAt}:{email:any,token:any,username:any,bio:any,image:any,id:any,createdAt:any,updatedAt:any}){
```

Comment:
utilize TypeScript instead of keeping using `any` types

Reply:
i just keep them until working with backend and know the type of returned data